### PR TITLE
Update readme, templates, and workflows

### PR DESCRIPTION
**Pull request by thespad**
[linuxserverurl]: https://linuxserver.io
[linuxserver.io][linuxserverurl]

------------------------------

 - [x] I have read the [contributing](https://github.com/linuxserver/docker-readarr/blob/main/.github/CONTRIBUTING.md) guideline and understand that I have made the correct modifications

------------------------------

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -80,26 +78,37 @@ Use the optional paths if you don't understand, or don't want hardlinks/atomic m
...
+>[!NOTE]
+>Unless a parameter is flaged as 'optional', it is *mandatory* and a value must be provided.
 
 ### docker-compose (recommended, [click here for more info](https://docs.linuxserver.io/general/docker-compose))
 
 ```yaml
 ---
 services:
   readarr:
-    image: lscr.io/linuxserver/readarr:develop
```
Comment (Roxedus) [inline README.md:93]: Do we want to break copy-pastes?

Comment (thespad) [reply README.md:93]: Happy to change it back, wasn't sure the best way to go with it.

**Comment by Roxedus** [comment]
Should probably also do the external files

File: README.md (hunk 1 above)
Comment (Roxedus) [reply README.md:93]: I think we should have it, its a big reason to why we populated the main branch at all

**Review by Roxedus — APPROVED** [review]

**Pull request merged**